AsyncDisplayKit/ASTableNode.m

```diff
@@ -77,6 +77,15 @@ - (void)didLoad
   }
 }
 
+- (void)updateCurrentRangeWithMode:(ASLayoutRangeMode)rangeMode
+{
+    if (!self.isNodeLoaded) {
+        return;
```

Comment:
Maybe instead of early return, just put the actual call in here, and remove the !